test/test_brokers.py

```diff
@@ -73,6 +78,45 @@ def test_fetch_brokers(self):
         if '.' not in host:
             self.fail("Didn't get a hostname from fetch_brokers()")
 
+    def _mocked_search(*args, **kwargs):
+        """Return values to mocked search call based on input."""
+
+        if (
```

Comment:
atleast it's PEP8 compliant....

Reply:
Yeah, it's not the nicest, but the LDAP queries aren't the shortest!